sdk/communication/azure-communication-common/src/main/java/com/azure/communication/common/PhoneNumberIdentifier.java

```diff
@@ -18,7 +18,7 @@
      * E.164 is a phone number formatted as +[CountryCode][AreaCode][LocalNumber] eg. "+18005555555"
```

Comment:
Creates a PhoneNumber object => Creates a PhoneNumberIdentifier object

Probably worthwhile to look for similar comments in the other classes too. "the string identifier representing the PhoneNumber" can stay unchanged, just add the *Identifier when referring to the exact type.